sites.json

```diff
@@ -3305,6 +3309,16 @@
     },
 
     {
+        "name": "Leetcode",
+        "url": "",
```

Comment:
Is there a link to the info that the account can't be deleted?

Reply:
I found no articles online, and the support team gave me no response :/

Reply:
Then I'd be fine with just a link to the website.